code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -55,6 +55,7 @@
 
 	//Hot simple_animal baby making vars
 	var/childtype = null
+	var/rarechildtype = null	//Special children
```

Comment:
Make childtype a weighted list and have the below bit pick from it instead. Would make it easier to add things down the line